### PR TITLE
Add Commit Messages  Guidelines

### LINKED ISSUE #3: Add 'Commit Messages' best practices section
I have a fairly large document prepped regarding this topic here: https://github.com/sscholle/best_practices/blob/master/Commit-Messages.MD

we should be able to use most of it

### PR DESCRIPTION
**Pull request by sscholle**
## Proposed changes

- Add Commit Messages  Guidelines

## Types of changes

What type of changes does your code introduce?
_Put an `x` in the boxes that apply, leave blank if none_

- [ ] Frontend
- [ ] Backend
- [ ] Bugfix
- [ ] New feature
- [ ] Component of Major Release
- [x] Project Management
- [ ] Quality of Life

## Relevant Issues

_List Issues below in format `#<ISSUE-NUMBER> [Full|Partial]`_
- Resolves #3 

## Description

Note that the guidelines are a copy-pasta and i've added them for review for any changes, feel free to mention anything.

### CODE REVIEW

File: commit_messages/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+* Capitalize the subject line and each paragraph
+* Use the imperative mood in the subject line
+* Wrap lines at 72 characters
+* Use the body to explain what and why you have done something. In most cases, you can leave out details about how a change has been made.
+
+### Information in commit messages
+* Describe why a change is being made.
+* How does it address the issue?
+* What effects does the patch have?
+* Do not assume the reviewer understands what the original problem was.
```
Comment (robguy21) [inline commit_messages/README.md]: Change this to:
```
* Add more context about why the code was changed, including business rules
```

File: commit_messages/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+* Do not assume the code is self-evident/self-documenting.
```
Comment (robguy21) [inline commit_messages/README.md]:
```
* Give a brief summary of the committed code
```

File: commit_messages/README.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+* Read the commit message to see if it hints at improved code structure.
```
Comment (robguy21) [inline commit_messages/README.md]: remove this

File: commit_messages/README.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+* The first commit line is the most important.
+* Describe any limitations of the current code.
+* Do not include patch set-specific comments.
+
+Details for each point and good commit message examples can be found on https://wiki.openstack.org/wiki/GitCommitMessages#Information_in_commit_messages
+
+### References in commit messages
```
Comment (robguy21) [inline commit_messages/README.md]: drop

**Comment by sscholle** [comment]
are we not adding the changes here before the merge?

**Pull request merged**